simulator/risc_v/t/unit_test.cpp

```diff
@@ -313,3 +313,78 @@ TEST_CASE("RISCV RV64 bfp")
     instr.execute();
     CHECK( instr.get_v_dst() == 0x0000'555C'5CCC'0000);
 }
+
+TEST_CASE ("RISCV RV32 clz")
+{
+    CHECK ( RISCVInstr<uint32>(0x60079793).get_disasm() == "clz $a5, $a5");
+    RISCVInstr<uint32> instr( "clz", 0);
+
+    instr.set_v_src( 0, 0);
+    instr.execute();
+    CHECK( instr.get_v_dst() == 32);
+
```

Comment:
These are separate tests cases. Please either separate them or use TestData as some of students used before.

Reply:
ok. 10 min.